docs/source/tutorials/QUBO_problem.ipynb

```diff
@@ -0,0 +1,580 @@
+{
```

Comment:
<p>it is not directly comparable in this figure, since the loss definition (different alpha) are different?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tencent-quantum-lab/tensorcircuit/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Hang on. I suddenly realise that they are comparable. In the ideal scenario, where the best result can be measured with 100% probability, their loss/cost should be the same.

Reply:
but except the perfect case, they are not comparable